### PR TITLE
move print pdf to fieldwork

### PR DESCRIPTION
**Pull request by lucienicolier**
add video

### CODE REVIEW

File: en/fieldwork/print-to-pdf.rst [hunk 1]
```diff
@@ -29,5 +29,13 @@ Done.
      :width: 500px
      :alt: PDF document
 
+Here is there a little video showing how it works:
```
Comment (m-kuhn) [inline en/fieldwork/print-to-pdf.rst:32]:
```suggestion
A video demonstration of how the complete workflow looks:
```

**Review by m-kuhn — APPROVED** [review]

**Comment by m-kuhn** [comment]
thanks

**Comment by lucienicolier** [comment]
Thanks to you @m-kuhn !

**Comment by lucienicolier** [comment]
Fixes #101

**Pull request merged**